extensions/cms/src/cmsResource/tree/cmsResourceTreeNode.ts

```diff
@@ -91,6 +91,7 @@ export class CmsResourceTreeNode extends CmsResourceTreeNodeBase {
 		item.contextValue = CmsResourceItemType.cmsNodeContainer;
 		item.id = this._id;
 		item.tooltip = this.description;
+		item.isServer = true;
 		item.iconPath = {
 			dark: this.appContext.extensionContext.asAbsolutePath('resources/light/centralmanagement_server.svg'),
```

Comment:
Just noticed this - have you reached out to design to get a dark mode icon for this?